### DISCUSSION TITLE
关于中文手写体识别的一些疑问

### DISCUSSION THREAD

**Discussion by Henryplay** [PaddlePaddle/PaddleOCR#12577] [Q&A]
我用中科院公开的手写体数据，把多个单字合成一行，使用识别模型训练没有精度。想问一下，这里合成一行时，是不是要有语义信息比较好呢，当前合成行数据的文字是随机挑选的。

**Comment by bi3qwq** [comment]
不清楚底层具体实现，不好回答

**Comment by justajustin** [comment]
这个问题和我遇到的是一样的，predict的时候会发现有精度，还不低，infer的时候全是乱码

**Comment by bi3qwq** [comment]
您的邮件我已经收到，这里是BI3QWQ。

**Comment by data-ant** [comment]
随机肯定可以的，学习的视觉特征与字符的对应关系

**Comment by Henryplay** [comment]
我是用开源的手写字，随机生成了70万张，跟印刷体在一块训练，手写效果不好

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2023年10月26日 15:45 |
| 收件人 | PaddlePaddle/PaddleOCR ***@***.***> |
| 抄送人 | Henry ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [PaddlePaddle/PaddleOCR] 关于中文手写体识别的一些疑问 (Issue #11068) |

随机肯定可以的，学习的视觉特征与字符的对应关系

**Reply by BeyondYourself** [reply]
用的哪个模型，介绍一下
